test/e2e_node/runner/run_e2e.go

```diff
@@ -393,30 +417,41 @@ func getGCEImages(imageRegex, project string, previousImages int) ([]string, err
 
 // Provision a gce instance using image and run the tests in archive against the instance.
 // Delete the instance afterward.
-func testImage(image *internalGCEImage, junitFilePrefix string) *TestResult {
-	host, err := createInstance(image)
+func testImage(imageConfig *internalGCEImage, junitFilePrefix string) *TestResult {
+	ginkgoFlagsStr := *ginkgoFlags
+	// Check whether the test is for benchmark.
+	if len(imageConfig.tests) > 0 {
```

Comment:
The logic is not very clear currently. It could be more generic.
Anyway, we can iterate in following PRs.